### PR TITLE
release: integration branch for all missing route scoping

### PR DESCRIPTION
**Pull request by soey**
sets up a release branch for route-scoping

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (roschaefer) [inline .github/workflows/ci.yml:5]: The meaning of this is to run CI on PRs targeting any `releases/**` branch. What about removing line 5 and run it on any PR? It's just CI not CD.

**Review by roschaefer — APPROVED** [review]
I would give this an "Approve" if the files changed would only contain `.github/workflows/ci.yml`. Maybe @soey can help me, why the other changes?

not sure

File: .github/workflows/ci.yml
Comment (mattwr18) [reply .github/workflows/ci.yml:5]: I agree with @roschaefer ... that is what I'd like.

**Comment by mattwr18** [comment]
> I would give this an "Approve" if the files changed would only contain `.github/workflows/ci.yml`. Maybe @soey can help me, why the other changes?
 
I think the purpose of this PR is to be able to manually test all changes together as they would be when deploying the feature, but still have the flexibility to have smaller PRs that can be merged into a base branch before that base branch is ultimately merged into main and deployed.

I find this PR helpful to be able to test, but I also would want to actually test the base branch after everything is merged into it because I think there will be merge conflicts that need to be worked through and sometimes we make mistakes, which can break things.

File: .github/workflows/ci.yml
Comment (soey) [reply .github/workflows/ci.yml]: done with: eca70b8a0a4d87a978e408db2a3da0b3e645646a

File: app/controllers/organizations_controller.rb
Comment (mattwr18) [inline app/controllers/organizations_controller.rb:17]:
```suggestion
 ```

I think this was copied from the PR that I suggested styling the organizations index page. I was playing around with something and it didn't reallly work as I expected. This method is never called and should be removed.

**Pull request merged**
